.github/workflows/ci.yaml

```diff
@@ -29,11 +29,13 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
+        pydantic-version: ["1", "2"]
     uses: ./.github/workflows/test.yaml
     with:
+      coverage: ${{ matrix.python-version == '3.11' && matrix.pydantic-version == '2' }}
```

Comment:
Is the the plan to only count coverage based on the v2 integration?

Reply:
for now. We can do something fancy with coverage combine at some point